ArtOfIllusion/src/artofillusion/image/ImageDetailsDialog.java

```diff
@@ -318,7 +318,7 @@ private void convertToLocal()
             return;
 
         String name = im.getName();
-        if (name.isEmpty());
+        if (name.isEmpty())
```

Comment:
BTW... This is a good example that no-brace code style is no good

Reply:
I fail to see the connection. It happens just as well with or without braces.